tasm/tiles/include/TiledVideoManager.h

```diff
@@ -5,12 +5,14 @@
 #include "TileLayout.h"
 #include "Video.h"
 #include <mutex>
+#include "LayoutDatabase.h"
 
 namespace tasm {
 class TiledVideoManager {
 public:
     TiledVideoManager(std::shared_ptr<TiledEntry> entry)
             : entry_(entry),
+              layouts_(LayoutDatabase::instance()),
```

Comment:
This is a nit and feel free to ignore, but `layouts_` makes me think that it is some sort of container (because of the plural-ness). What about `layoutDB_`?